harness/determined/cli/agent.py

```diff
@@ -14,6 +14,8 @@
 from determined.common.api import authentication, bindings
 from determined.common.declarative_argparse import Arg, Cmd, Group
 
+NO_PERMISSIONS = "NO PERMISSIONS"
```

Comment:
Are you still using this?

Reply:
Yea! Gets used on line 99 in `get_task_name`.